### PR TITLE
Add udf citus_get_active_worker_nodes

### PR DESCRIPTION
**Pull request by agedemenli**
DESCRIPTION: Introduces new UDF citus_get_active_worker_nodes

fixes: #4828

### CODE REVIEW

**Comment by SaitTalhaNisanci** [comment]
In DESCRIPTION we use present tense, so I changed it to `Introduces`.

File: src/backend/distributed/operations/node_protocol.c [hunk 1]
```diff
@@ -85,6 +85,7 @@ PG_FUNCTION_INFO_V1(master_get_table_ddl_events);
 PG_FUNCTION_INFO_V1(master_get_new_shardid);
 PG_FUNCTION_INFO_V1(master_get_new_placementid);
 PG_FUNCTION_INFO_V1(master_get_active_worker_nodes);
+PG_FUNCTION_INFO_V1(citus_get_active_worker_nodes);
```
Comment (SaitTalhaNisanci) [inline src/backend/distributed/operations/node_protocol.c:88]: it feels a bit odd that this is in between all the other `master` UDFs

File: src/backend/distributed/sql/citus--10.0-2--10.0-3.sql [hunk 2]
```diff
@@ -8,3 +8,11 @@ RETURNS VOID
     AS 'MODULE_PATHNAME', $$citus_update_table_statistics$$;
 COMMENT ON FUNCTION pg_catalog.master_update_table_statistics(regclass)
 	IS 'updates shard statistics of the given table';
+
+CREATE FUNCTION pg_catalog.citus_get_active_worker_nodes(OUT node_name text, OUT node_port bigint)
```
Comment (SaitTalhaNisanci) [inline src/backend/distributed/sql/citus--10.0-2--10.0-3.sql:12]: maybe `CREATE OR REPLACE FUNCTION` ..

File: src/test/regress/expected/multi_extension.out [hunk 3]
```diff
@@ -543,9 +543,10 @@ SELECT * FROM print_extension_changes();
 -- Snapshot of state at 10.1-1
 ALTER EXTENSION citus UPDATE TO '10.1-1';
 SELECT * FROM print_extension_changes();
- previous_object | current_object
+ previous_object |              current_object
```
Comment (onderkalaci) [inline src/test/regress/expected/multi_extension.out:546]: why do we see this in `'10.1-1';`?

Comment (onurctirtir) [reply src/test/regress/expected/multi_extension.out:563]: we already talked in teams but sharing the link for reference
https://github.com/citusdata/citus/pull/4747/commits/dcc020760519edda38753d176ff3c9ac718640cf

**Review by SaitTalhaNisanci — APPROVED** [review]

**Pull request merged**